static/app/components/quickTrace/index.tsx

```diff
@@ -79,12 +79,20 @@ export default function QuickTrace({
 }: QuickTraceProps) {
   let parsedQuickTrace;
   try {
-    parsedQuickTrace = parseQuickTrace(quickTrace, event, organization);
+    if (quickTrace.orphanErrors && quickTrace.orphanErrors.length > 0) {
```

Comment:
What if there are orphanErrors but this event isn't one of them? Should `parseQuickTrace` still be tried if the find condition failed?

Reply:
Good point. We send a flattened version of orphanerrors down to quick trace. So that array if non-empty will have just one error in ti. Actually we don't need an array here. I can just pass an orphanError: TraceError here and check if it exists. I'll make the change.